src/mon/MDSMonitor.cc

```diff
@@ -1564,99 +1551,6 @@ int MDSMonitor::filesystem_command(
   return r;
 }
 
-/**
- * Helper to legacy_filesystem_command
- */
-void MDSMonitor::modify_legacy_filesystem(
```

Comment:
This is rude, but we have not documented these commands for some time. Now that we're changing their behavior, it might make sense to finally rip them out.